frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -358,6 +358,22 @@ export default function SurveyEdit(): JSX.Element {
                                                                                   textPlaceholder="ex: We really appreciate it."
                                                                               />
                                                                           </LemonField.Pure>
+                                                                          <LemonField.Pure label="Button text">
```

Comment:
Add a top margin please. Or add a single `space-y-2` to the parent element to control the spacing between fields.

```suggestion
                                                                          <LemonField.Pure className="mt-2" label="Button text">
```